HakawaiDemoSwift/HakawaiDemoSwift/Mentions/MentionsDemoViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  MentionsDemoViewController.swift
+//  HakawaiDemoSwift
+//
+//  Copyright (c) 2014 LinkedIn Corp. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+//  the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+
+import UIKit
+import Hakawai
+
+class MentionsDemoViewController: UIViewController {
+    
+    //MARK: - Outlets
+    @IBOutlet weak var textView: HKWTextView!
+    @IBOutlet weak var mentionsListButton: UIButton!
+    
+    //
+    private var plugin: HKWMentionsPlugin!
```

Comment:
Please review and apply LinkedIn's Swift Code Style - https://github.com/linkedin/swift-style-guide

For example here you should not use implicitly unwrapped optional, and make it a regular optional.